src/JsonConfig.cs

```diff
@@ -3,14 +3,14 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 using System;
-using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 using System.ComponentModel;
 using System.IO;
-using System.Timers;
 using Newtonsoft.Json;
+using System.Windows.Forms;
+using Timer = System.Timers.Timer;

```

Comment:
Minor stylistic detail, but would prefer if `MessageBox` below is referenced as `System.Windows.Forms.MessageBox`. This would make this import of `System.Windows.Forms` and the Timer override unnecessary.

Reply:
Yeah I noticed this later. Will do.